addons/website_slides/models/slide_channel.py

```diff
@@ -95,6 +95,8 @@ def _default_access_token(self):
         string="Course type", default="documentation", required=True)
     sequence = fields.Integer(default=10, help='Display order')
     user_id = fields.Many2one('res.users', string='Responsible', default=lambda self: self.env.uid)
+    responsible_name = fields.Char('Responsible Name', related='user_id.name')
+    responsible_email = fields.Char('Responsible Email', related='user_id.email')
```

Comment:
I wonder if we should name those `user_name` and `user_email` instead to reflect the fact that it's based on the `user_id` field.
Leave it like that for now, I'd like to have @tde-banana-odoo opinion on this.